website/server/libs/cron.js

```diff
@@ -347,7 +393,7 @@ export function cron (options = {}) {
 
   // Login Incentives
   user.loginIncentives++;
-  user.addNotification('LOGIN_INCENTIVE', {});
+  awardLoginIncentives(user);
```

Comment:
@Alys I'm not saying that it's not a good idea but just that if we add another save location for the user then we either save the lastCron date the first time we save the user, which means that the parts that didn't run will be skipped or we do not save lastCron which means we could have a partial double cron in case of errors. Unfortunately there's no 100% correct way to do this without transactions